lang/ruby/test/test_schema.rb

```diff
@@ -160,4 +160,101 @@ def test_to_avro_handles_falsey_defaults
       ]
     }
   end
+
+  def test_support_for_doc_attribute
```

Comment:
I have an RSpec background. I don't have `test-unit` experience. Should these all be separate tests? How would you structure these?